presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -298,322 +314,106 @@ public static String graphvizDistributedPlan(SubPlan plan)
         return GraphvizPrinter.printDistributed(plan);
```

Comment:
Do you plan to add GraphvizRenderer in a future PR?

Reply:
I don't have that PR planned, but yes that would be the way to go.